python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -170,6 +171,7 @@ def __init__(
         self.done_task = utils.make_done_future()
         self.task = utils.make_done_future()
         self.config()
+        self.log.info("SerialTemperatureScanner configured.")
```

Comment:
Line 169 `self.start_task = utils.make_done_future()` may be removed.

Reply:
I am not happy about starting everything from the constructor. We will need to talk about that as well.

Reply:
Line 171 `self.task = utils.make_done_future()` please provide a more meaningful name for that variable.

Reply:
Line 173 `self.config()` is called after all tasks have been started. Is that really what you want? There is a TODO in the config method indicating that some day we will get the sensor config from ESS so that indicates that the config method should be called before starting the tasks.